data/random-teams.js

```diff
@@ -1392,9 +1392,9 @@ class RandomTeams extends Dex.ModdedDex {
 		} else if (hasMove['magikarpsrevenge']) {
 			// PoTD Magikarp
 			item = 'Choice Band';
-		} else if (template.species === 'Rotom-Fan') {
+		} else if (template.species === 'Stunfisk' && !teamDetails['sand']) {
 			// This is just to amuse Zarel
```

Comment:
Wow preserving the comment... okay fine.

I would probably set `ability = 'Limber'` here instead of up there.